docker/Dockerfile

```diff
@@ -44,6 +44,12 @@ RUN git clone https://gitlab.archlinux.org/archlinux/archlinux-keyring && \
     make build && \
     make PREFIX=/usr KEYRING_TARGET_DIR=/usr/share/keyrings/ DESTDIR=/arch-keyring install
 
+# Install Debian ports keyring
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        debian-ports-archive-keyring && \
```

Comment:
This needs to go into the runtime dependencies section

Reply:
Updated accordingly.